### DISCUSSION TITLE
how to customize i3's ordering of displays/monitors for move/focus

### DISCUSSION THREAD

**Discussion by m10d** [i3/i3#5677] [Q&A]
I have two 4k screens, the one on the left `DP-4` landscape, and on the right `DP-0`  portrait.  I've added a drawing tablet `HDMI-0`, video-card wise "in between them" - but I want to change which display i3 considers left/right of which.  They are laid out this way due to a graphics stack issue with "total screen area" which includes empty space (not relevant to this issue, I'm just clarifying my setup)

A picture is worth a ... 
![monitors 2023-09-19_13-29](https://github.com/i3/i3/assets/59897595/9f781ea9-12a3-4191-b44b-9bb858047ee7)

**Comment by orestisfl** [comment]
Silly suggestion but moving HDMI-0 one pixel to the left will do what you want and them mouse will still travel between monitors. Not sure if there are other downsides.

Otherwise, https://github.com/i3/i3/issues/5517 is related
